VEHICLE/SetVehicleOnGroundProperly.md

```diff
@@ -17,6 +17,8 @@ NativeDB Added Parameter 2: float p1
 ```
```

Comment:
Change this to (just copy the markdown):

`````

Sets a vehicle on the ground on all wheels.

### Newly added parameters:

```
NativeDB Added Parameter 2: float p1
```

* **p1**:  Might correlate to the vehicle size that needs to be set on the ground. Values used in scripts: 0f, 1f, 5f, 7f, 7.5f(for bikes apparently), 8f.

`````

Removed the return bit since it was somewhat redundant noticing how it's already mentioned in the 'return value' section.